genai-on-vertex-ai/vertex_evaluation_services/gemini-curate-evaluation-data/curate_new_evals.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<p>Could we make the variables a bit more clear -</p><p>Project could mean PROJECT_ID/NUMBER. Standard notation usage is preferred.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>